### ISSUE TITLE
test set

### ISSUE THREAD

**Issue by shikun1408** [divyanshu-talwar/AutoImpute#9]
Hello, author, what is the ratio of the training set to the test set in your autoimpute paper? I didn’t find it in your code. I only found the ratio of the mask. Do you need the ratio of train and test? Set it yourself. Thank you very much! Looking forward to your reply!

Best wish！

**Comment by divyanshu-talwar** [comment]
Hey,

The `masking_percentage` parameter determines the train-test split of the input dataset. For instance, if the `masking_percentage` parameter is set to 20%, the script would mask 20% of the input data during the training process, essentially having a 80-20 train-test split.

**Comment by shikun1408** [comment]
Because your reply allowed me to solve the problem. I am so gald to received to your reply,thank you very&nbsp; much!


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "divyanshu-talwar/AutoImpute"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年1月18日(星期一) 下午5:53
收件人:&nbsp;"divyanshu-talwar/AutoImpute"<AutoImpute@noreply.github.com&gt;;
抄送:&nbsp;"stone"<759188075@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [divyanshu-talwar/AutoImpute] test set (#9)





 
Hey,
 
The masking_percentage parameter determines the train-test split of the input dataset. For instance, if the masking_percentage parameter is set to 20%, the script would mask 20% of the input data during the training process, essentially having a 80-20 train-test split.
